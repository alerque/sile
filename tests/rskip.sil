\begin[papersize=a5,class=book]{document}
\use[module=packages.lorem]
\set[parameter=document.rskip,value=20mm]
\lorem[words=50]
\end{document}
