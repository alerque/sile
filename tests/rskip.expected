Set paper size 	419.5275636	595.275597
Begin page
Mx 	78.7339
My 	76.6203
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	106.6871
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	135.4899
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	161.0212
T	86 76 87 w=10.0146	(sit)
Mx 	174.6385
T	68 80 72 87 w=20.6836	(amet)
Mx 	198.9247
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	246.6875
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	293.2882
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	314.3322
T	86 72 71 w=13.6816	(sed)
Mx 	58.7339
My 	88.6203
T	71 76 68 80 w=20.5322	(diam)
Mx 	81.7349
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	118.5299
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	150.5495
T	87 72 80 83 82 85 w=30.3809	(tempor)
Mx 	183.3992
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	221.1952
T	88 87 w=8.7402	(ut)
Mx 	232.4043
T	79 68 69 82 85 72 w=25.9863	(labore)
Mx 	260.8594
T	72 87 w=8.0615	(et)
Mx 	271.3898
T	71 82 79 82 85 72 w=26.5479	(dolore)
Mx 	300.4064
T	80 68 74 81 68 w=27.6074	(magna)
Mx 	58.7339
My 	100.6203
T	68 79 76 84 88 92 68 80 w=38.0127	(aliquyam)
Mx 	99.6836
T	72 85 68 87 w=16.6113	(erat)
Mx 	119.2319
T	86 72 71 w=13.6816	(sed)
Mx 	135.8506
T	71 76 68 80 w=20.5322	(diam)
Mx 	159.3199
T	89 82 79 88 83 87 88 68 w=36.5967	(voluptua)
Mx 	198.8536
T	68 87 w=8.0322	(at)
Mx 	209.8228
T	89 72 85 82 w=18.5400	(vero)
Mx 	231.2999
T	72 82 86 w=13.5107	(eos)
Mx 	247.7477
T	72 87 w=8.0615	(et)
Mx 	258.7462
T	68 70 70 88 86 68 80 w=35.1123	(accusam)
Mx 	296.7956
T	72 87 w=8.0615	(et)
Mx 	307.7941
T	77 88 86 87 82 w=20.2197	(justo)
Mx 	58.7339
My 	112.6203
T	71 88 82 w=15.5273	(duo)
Mx 	76.5035
T	71 82 79 82 85 72 86 w=30.4102	(dolores)
Mx 	109.1560
T	72 87 w=8.0615	(et)
Mx 	119.4599
T	72 68 w=9.2090	(ea)
Mx 	130.9112
T	85 72 69 88 80 w=26.9873	(rebum)
Mx 	160.1409
T	86 87 72 87 w=15.3662	(stet)
Mx 	177.7494
T	70 79 76 87 68 w=17.8223	(clita)
Mx 	197.8140
T	78 68 86 71 w=18.5303	(kasd)
Mx 	218.5866
T	74 88 69 72 85 74 85 72 81 w=42.8076	(gubergren)
Mx 	263.6366
T	81 82 w=10.5469	(no)
Mx 	276.4258
T	86 72 68 w=13.0713	(sea)
Mx 	291.7394
T	87 68 78 76 80 68 87 68 w=36.2744	(takimata)
Mx 	58.7339
My 	124.6203
T	86 68 81 70 87 88 86 w=30.9424	(sanctus)
Mx 	92.3307
T	72 86 87 w=11.9238	(est)
Mx 	106.9091
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	133.9142
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	161.7689
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	186.3521
T	86 76 87 w=10.0146	(sit)
Mx 	199.0212
T	68 80 72 87 w=20.6836	(amet)
Mx 	219.3741
My 	519.8019
T	20 w=4.6924	(1)
End page
Finish
